### ISSUE TITLE
OSGI manifest is missing for si-units-java8

### ISSUE THREAD

**Issue by ChristianLutz** [unitsofmeasurement/si-units#55]
Hello,

I don't know what the reason is but the current version (0.9) of si-units-java8 doesn't contain OSGI information though the maven-bundle-plugin is used within the [pom.xml](https://github.com/unitsofmeasurement/si-units/blob/master/units-java8/pom.xml) 

Current manifest (Found [in](https://mvnrepository.com/artifact/si.uom/si-quantity/0.9))
```
Manifest-Version: 1.0
Implementation-Title: SI Units for Java SE 8
Implementation-Version: 0.9
Built-By: werner
Specification-Vendor: Units of Measurement
Implementation-Vendor-Id: si.uom
Specification-Title: Units of Measurement Systems
Created-By: Apache Maven 3.3.9
Build-Jdk: 1.8.0_162
Implementation-URL: http://unitsofmeasurement.github.io/
Specification-Version: 0.9
Implementation-Vendor: Unit-API contributors
```

Also the issue #25 has been closed but the pom doesn't reflect the suggested changes. Any reason?

kind regards

**Comment by keilw** [comment]
Have to check, if there's a difference because the main (indriya-based, the next gen RI which is already implementing JSR 363 now) implementation reads

```
Bundle-Description: Units of Measurement SI (Système International d'
 Unités)
Bundle-DocURL: http://unitsofmeasurement.github.io/
Bundle-License: http://opensource.org/licenses/BSD-3-Clause
Bundle-ManifestVersion: 2
Bundle-Name: SI Units
Bundle-SymbolicName: si.uom.si-units
Bundle-Vendor: Units of Measurement
Bundle-Version: 0.9.0
Created-By: Apache Maven Bundle Plugin
Export-Package: si.uom;uses:="javax.measure,javax.measure.quantity,si.
 uom.quantity,tec.units.indriya,tec.units.indriya.unit";version="0.9.0
 "
Implementation-Title: SI Units
Implementation-URL: http://unitsofmeasurement.github.io/
Implementation-Vendor: Unit-API contributors
Implementation-Vendor-Id: si.uom
Implementation-Version: 0.9
Import-Package: javax.measure;version="[1.0,2)",javax.measure.quantity
 ;version="[1.0,2)",javax.measure.spi;version="[1.0,2)",si.uom,si.uom.
 quantity;version="[0.9,1)",tec.units.indriya;version="[1.0,2)",tec.un
 its.indriya.format;version="[1.0,2)",tec.units.indriya.function;versi
 on="[1.0,2)",tec.units.indriya.quantity;version="[1.0,2)",tec.units.i
 ndriya.spi;version="[1.0,2)",tec.units.indriya.unit;version="[1.0,2)"
 ,tec.uom.lib.common.function;version="[1.0,2)"
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
Specification-Title: Units of Measurement Systems
Specification-Vendor: Units of Measurement
Specification-Version: 0.9
Tool: Bnd-3.4.0.201707252008
```

**Comment by keilw** [comment]
It works for the Indriya based module, but not the (old) Java SE 8 one. @duckAsteroid any idea, what could go wrong in `si-units-java8`?

**Comment by keilw** [comment]
@ChristianLutz It looks completely unsolvable I'm afraid. No clue why the Felix plugin is ignored for that module only  and all others work? I analyzed the POM and found no reason why it's ignored either.

**Comment by keilw** [comment]
Seems the "bundle" package type was missing, but I gave the `bnd-plugin` a try and it generates even a little more details than Felix. Will be solved in 1.0

**Comment by ChristianLutz** [comment]
perfect, thank you.
